recipes/colmap/meta.yaml

```
@@ -0,0 +1,99 @@
+{% set name = "colmap" %}
+{% set version = "3.6" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/{{ name }}/{{ name }}/archive/{{ version }}.tar.gz
+    sha256: ea22d026a3b13d339551319f4562fe1471b7232433695970704669755031a192
+    patches:
+      - fix_build.patch
+
+build:
+  number: 0
+  skip: true  # [win]
+  run_exports:
+    - {{ pin_subpackage('colmap', max_pin='x.x') }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+    - cmake
+    - pkg-config
+    - ninja
+    - llvm-openmp                    # [osx]
+    - libgomp                        # [linux]
+    # x11 stuff
+    - {{ cdt('mesa-libgl-devel') }}  # [unix]
+    - {{ cdt('mesa-dri-drivers') }}  # [unix]
+    - {{ cdt('libselinux') }}        # [linux]
+    - {{ cdt('libxcb') }}            # [linux]
+    - {{ cdt('libxdamage') }}        # [linux]
+    - {{ cdt('libxfixes') }}         # [linux]
+    - {{ cdt('libxxf86vm') }}        # [linux]
+    - {{ cdt('libxext') }}           # [linux]
+    - {{ cdt('libxau') }}            # [linux]
+    - {{ cdt('expat-devel') }}       # [linux]
+  host:
+    # x11 stuff
+    - xorg-libxext                   # [unix]
+    - xorg-libxdmcp                  # [unix]
+    - xorg-libxau                    # [unix]
+    - libxcb                         # [unix]
+    - xorg-libxfixes                 # [unix]
+    - xorg-libxrender                # [unix]
+    - xorg-libx11                    # [unix]
+    - xorg-libice                    # [unix]
+    - xorg-libsm                     # [unix]
+    - libglu                         # [linux]
+    # deps
+    - boost-cpp
+    - suitesparse
+    - eigen
+    - freeimage
+    - glog
+    - gflags
+    - glew
+    - cgal-cpp
+    - ceres-solver
+    - qt
+    - git
+    - flann
+    - sqlite
+    - vlfeat
+    - gmp                            # [unix]
+
+  run:
+    - boost-cpp
+    - vlfeat
+    - flann
+    - xorg-libxext                   # [unix]
+    - xorg-libxdmcp                  # [unix]
+    - xorg-libxau                    # [unix]
+    - libxcb                         # [unix]
+    - xorg-libxfixes                 # [unix]
+    - xorg-libxrender                # [unix]
+    - xorg-libx11                    # [unix]
+    - xorg-libice                    # [unix]
+    - xorg-libsm                     # [unix]
+    - libglu                         # [linux]
+
+test:
+  commands:
+    - if exist %PREFIX%\\Library\\bin\\colmap.exe (exit 0) else (exit 1)  # [win]
+    - test -f $PREFIX/bin/colmap  # [unix]
+    - test -f $PREFIX/lib/colmap/libcolmap.a  # [unix]
+    - test -f $PREFIX/share/colmap/COLMAPConfig.cmake  # [unix]
+
+about:
+  home: https://colmap.github.io/
+  license:  BSD-3-Clause
+  license_file: COPYING.txt
```

You should list all the licenses here
```suggestion
  license_file:
   - COPYING.txt
   - …
```